src/jit/emit.cpp

```diff
@@ -3341,23 +3341,23 @@ void emitter::emitDispIGlist(bool verbose)
 void emitter::emitDispGCinfo()
 {
     printf("Emitter GC tracking info:");
-    printf("\n  emitPrevGCrefVars(0x%p)=%016llX ", dspPtr(&emitPrevGCrefVars), emitPrevGCrefVars);
+    // printf("\n  emitPrevGCrefVars(0x%p)=%016llX ", dspPtr(&emitPrevGCrefVars), emitPrevGCrefVars);
     dumpConvertedVarSet(emitComp, emitPrevGCrefVars);
```

Comment:
Why is this commented out? I guess because these are bitsets that aren't necessarily pointers. You should at least leave the:
```
printf("\n  emitPrevGCrefVars ");
```